Pagination: describe API (#6707)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxPagination/1 Configuration/itemCount.md

---
id: dxPagination.Options.itemCount
type: Number
default: 1
---
---
##### shortDescription
Specifies the total number of items.

---

> `itemCount` is the main configuration setting of the component. Specify the total number of items to ensure that the Pagination component functions properly.

Reviewer: ```suggestion
`itemCount` is the main configuration setting of the component. Specify the total number of items to ensure the Pagination component functions properly.
```

> `itemCount` is the main configuration setting of the component. Specify the total number of items that should be paginated for Pagination to function properly.

Reviewer 2: "Pagination" is not a great name to use on its own. I can say "Pager works correctly". "Pager" is countable - one thing. But "Pagination" is almost like an action, it's definitely uncountable. If you say "Pagination works correctly", it feels like you refer to the "process of pagination" not to "Pagination the component". So I would refer to the component as "the Pagination component". 

I also don't want to use words "paginated" and "Pagination" so close together. 

```suggestion
`itemCount` is the main configuration setting of the component. Specify the total number of items to ensure that the Pagination component functions properly.
```
